test/0.8.9/oracle/hash-consensus-access-control.test.js

```diff
@@ -170,18 +172,20 @@ contract('HashConsensus', ([admin, account1, account2, member1, member2, strange
 
     context('setReportProcessor', () => {
       it('should revert without MANAGE_REPORT_PROCESSOR_ROLE role', async () => {
+        const reportProcessor2 = await MockReportProcessor.new(CONSENSUS_VERSION, { from: admin })
```

Comment:
why not `reportProcessor` ? Without 2

Reply:
Because first is already sewed into initial consensus deploy